pkg/query-service/app/logs/parser_test.go

```diff
@@ -372,7 +380,10 @@ var generateSQLQueryTestCases = []struct {
 func TestGenerateSQLQuery(t *testing.T) {
 	for _, test := range generateSQLQueryTestCases {
 		Convey("testGenerateSQL", t, func() {
-			res, _ := GenerateSQLWhere(&generateSQLQueryFields, &test.Filter)
+			res, _, err := GenerateSQLWhere(&generateSQLQueryFields, &test.Filter)
```

Comment:
Well, this mismatch number of args means that tests are not run currently. @nityanandagohain, please make them run as a part of the workflow.